eox_tenant/edxapp_wrapper/backends/bearer_authentication_m_v1.py

```diff
@@ -0,0 +1,16 @@
+"""Backend for authentication.
```

Comment:
Is this backend compatible with Lilac? Could we name it ..._l_v1 so that we create this functionality already expecting that limonero can use it?

Reply:
Yes you are right, it is compatible with Lilac
Thanks @felipemontoya, I changed the file name

Reply:
actually, It's compatible with Juniper as well